packages/ra-supabase/README.md

```diff
@@ -86,6 +89,18 @@ export const MyAdmin = () => (
 
 You must wrap your `<Admin>` inside a `<BrowserRouter>` as supabase use hash parameters for passing authentication tokens.
 
+### Using Hash Router
+
+Supabase uses URL hash links for its redirections. This can cause conflicts if you use a HashRouter. For this reason, we recommend using the BrowserRouter.
+
+If you want to use the HashRouter, you'll need to modify the code.
+
+1. Create a custom `auth-callback` folder inside your public folder.
+2. Create an `index.html` file inside the `auth-callback`. This file will intercept the supabase redirect and rewrite the URL to prevent conflicts with the HashRouter. For example, see `packages/demo/public/auth-callback/index.html`.
+3. Remove `BrowserRouter` from your `App.ts`
+4. In **URL Configuration**, add the following URL in the **Redirect URLs** section: `YOUR_APPLICATION_URL/auth-callback/index.html`
```

Comment:
> In **URL Configuration**

Where is that configuration found? In the supabase settings? If so, please mention it